### PR TITLE
Fix Lenovo Thinkpad T14s not powering off

### PR DESCRIPTION
**Pull request by tobias-kuendig**
###### Description of changes

My Lenovo Thinkpad T14s Gen 4 does not propery power off after shutdown. The display goes blank but the CPU runs very hot and the fan starts to spin up. In this state, the battery is drained within around an hour.

Adding the `apm` parameter fixes this behaviour.

Maybe someone else can chime in if they had similar problems.

###### Things done

- [X] Tested the changes in your own NixOS Configuration
- [ ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by pinage404** [comment]
sometimes, i have the same issue on Framework 16

File: lenovo/thinkpad/t14s/amd/gen4/default.nix [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{ lib, pkgs, ... }:
```
Comment (Mic92) [inline lenovo/thinkpad/t14s/amd/gen4/default.nix:1]: Can you add this model to flake.nix and the table in the README?

Comment (tobias-kuendig) [reply lenovo/thinkpad/t14s/amd/gen4/default.nix:1]: Done!

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**